package.json

```diff
@@ -1,7 +1,7 @@
 {
   "dependencies": {
     "babel-cli": "^6.16.0",
-    "babel-preset-steelbrain": "^5.0.1",
+    "@mcro/preset-motion": "5.0.2",
```

Comment:
Any ideas why this was reverted? Can you show me the error the other preset was causing?